package/endpoint/data_stream/process/fields/fields.yml

```
@@ -997,6 +997,18 @@
       type: double
       description: Number of seconds since the process's name was modified. This information can come from the NTFS MFT. This number may be negative if the file's timestamp is in the future.
       default_field: false
+    - name: Ext.script.content
+      level: custom
+      type: keyword
+      ignore_above: 1024
```

```suggestion
      type: match_only_text
```

otherwise what's the purpose of keyword searching for the first 1KB of the script content? Was it really intended?